node/cypress/integration/actionButtons.js

```diff
@@ -0,0 +1,39 @@
+describe('defaultButton.shinyInput()', () => {
+  it('contents gets sent to server and displayed in an output', () => {
```

Comment:
Please change to `'clicks work'` or something else that describes what the test does.